components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/util/OAuth2Util.java

```diff
@@ -2016,7 +2016,14 @@ public static OauthTokenIssuer getOAuthTokenIssuerForOAuthApp(OAuthAppDO appDO)
                 }
             }
         }
-        return (claimConfig.toString().split(","));
+
+        String[] claim;
+        if (claimConfig.length() > 0) {
+            claim = claimConfig.toString().split(",");
+        } else {
+            claim = new String[0];
+        }
+        return (claim);
```

Comment:
shall we remove the redundant brackets here

Reply:
Addressed with commit 3e36e5f40f310a3901eed2b66fc54932223cf789